module/importer-magento-1/src/Infrastructure/Processor/Step/Magento1ConfigurableProductProcessor.php

```diff
@@ -178,24 +181,26 @@ private function getRecord(ProductModel $product, Transformer $transformer, Mage
     }
 
     /**
-     * @param array $default
+     * @param ProductModel $productModel
+     * @param array        $default
      *
      * @return null|string
      */
-    private function getBindings(array $default): ?string
+    private function getBindings(ProductModel $productModel, array $default): ?string
     {
         $result = [];
+        $bindings = [];
 
-        if (array_key_exists('bindings', $default)) {
+        if (array_key_exists('bindings', $default) && $default['bindings'] !== null) {
```

Comment:
```suggestion
        if ($bindings = $default['bindings'] ?? null) {
```
It seems way easier to read but up to you